### PR TITLE
Mdlsite 4220

### PR DESCRIPTION
**Pull request by stronk7**
Not much to say. It just adds a commit to the just created branch and includes some text/color cleanup. best to understand it by using it (./prerelease --type major).

Ciao :-)

### CODE REVIEW

File: adjusttravis.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+    throw new Exception('Invalid path given.', __LINE__);
+}
+
+function validate_travis_file($contents, $branch) {
+    // Some random bits to look for.
+    $hasmatrix = strpos($contents, 'matrix:') !== false;
+    $hasallowfailures = strpos($contents, '    allow_failures:') !== false;
+    $haslocalci = strpos($contents, 'moodlehq/moodle-local_ci') !== false;
+    $hasphplint = strpos($contents, 'php_lint/php_lint.sh') !== false;
```
Comment (andrewnicols) [inline adjusttravis.php]: I'm not sure about this bit - later on we *may* decide that it's better to add a script to local_ci for all of travis stuff which runs the lint, and all other bits which we are capable of running. Thoughts?

**Comment by stronk7** [comment]
Thanks Andrew, I've simply deleted that test, agree it may fail in the future, so better look for fixed, known to exist, contents. Good point.

**Comment by andrewnicols** [comment]
Thanks Eloy - looks good. Had to google tput and friends ;)

**Comment by stronk7** [comment]
Thanks! Resolving https://tracker.moodle.org/browse/MDLSITE-4220...

**Pull request merged**